frontend/.eslintrc.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  'env': {
+    'browser': true,
+    'node': true,
+    'es2021': true
+  },
+  'extends': [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended'
+  ],
+  'parser': '@typescript-eslint/parser',
+  'parserOptions': {
+    'ecmaFeatures': {
+      'jsx': true
+    },
+    'ecmaVersion': 'latest',
+    'sourceType': 'module'
+  },
+  'plugins': [
+    'react',
+    '@typescript-eslint'
+  ],
+  'settings': {
+    'react': {
+      'version': 'detect'
+    }
+  },
+  'rules': {
+    'indent': [
+      'error',
+      2
+    ],
+    'linebreak-style': [
+      'error',
+      'unix'
+    ],
+    'quotes': [
+      'error',
+      'single'
+    ],
+    'semi': [
+      'error',
+      'always'
+    ]
```

Comment:
Not sure about enforcing this, but we were kinda using them sporadically, I guess I figured we may as well be consistent.

I'm hesistant to adopt `standard` right now because the `standard` + ESLint + TypeScript + React stuff is a little behind. I figured we could just define our own rule set as we go for now.